snap7/client_async.py

```diff
@@ -1,18 +1,30 @@
 """
 Snap7 async client used for connection to a siemens7 server.
 """
-from ctypes import c_int, byref
+from ctypes import c_int, byref, c_byte
 
 import logging
 import asyncio
 
 import snap7
 from snap7.common import check_error
 from .client import Client
+from snap7.snap7types import buffer_type, buffer_size
+
 
 logger = logging.getLogger(__name__)
 
 
+def error_wrap(func):
```

Comment:
why not import this from the other module?

Reply:
It is not imported in server.py, client.py and partner.py as well from anywhere.
Would util.py maybe a good place to store it there and then import it in these files?

Reply:
ah you are right, not sure why. Yeah, i think it could be a good one to centralize, but it is not crazy important.

Reply:
util.py seems to be for something else. I would leave it where it is at the moment and we should tidy it up in a later PR.